src/components/table-cell/GenericCell.tsx

```diff
@@ -29,11 +29,11 @@ export default function GenericCell({
 
   const content = useMemo(() => {
     if (value === null) {
-      return <span className="text-gray-500">NULL</span>;
+      return <span className="text-gray-500 float-right">NULL</span>;

```

Comment:
The alignment of the NULL or DEFAULT should be based on the data type.
- Number data type means align right. I believe we need to pass the header into the GenericCell to deal with it

Reply:
Ahh, I should have read the issue properly. 

In that case, what about the column header? Should the heading following this alignment rule too? Or is it an exception to this rule?

Reply:
The column header does not need to follow this rule.

Reply:
got will work on it